tasks.py

```diff
@@ -66,15 +66,20 @@ def flake8(context):
 
 
 @task(aliases=["test", "tests"])
-def pytest(context):
+def pytest(context, color="yes", external=False):
```

Comment:
I'd add some explanation of that argument. Remember that for invoke tasks, the way to add documentation so that it is nicely formatted in `invoke help` is to add it to the decorator.